components/contactsList/styles.js

```diff
@@ -19,10 +19,23 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import { createStyles } from "@solid/lit-prism-patterns";
+import { createStyles, table } from "@solid/lit-prism-patterns";
 
 const styles = (theme) => {
-  return createStyles(theme, ["container", "table"], {
+  const tableStyles = table.styles(theme);
+  return createStyles(theme, ["container", "table", "icons"], {
+    table: {
+      ...tableStyles.table,
+      "& tbody": tableStyles.table__body,
```

Comment:
I realize now that we might just as well make this default behavior in Prism (e.g. have rules for both `table__body` and `& tbody` that points to the same rules). That way we don't have to repeat this block of code for every table in PB.

(I initially don't want to do this, as it makes CSS specificity harder for later overrides, but I think it's preferrable in this case.)